packages/repository-tests/src/crud/relations/acceptance/has-many.relation.acceptance.ts

```diff
@@ -221,7 +221,7 @@ export function hasManyRelationAcceptance(
           ],
         }),
       ).to.be.rejectedWith(
-        'Navigational properties are not allowed in model data (model "Customer" property "orders")',
+        'Navigational properties are not allowed in model data (model "Customer" property "orders"). Please remove it or make sure the relation name is not the same as the property name in belongsTo relation.',
```

Comment:
This line is too long and difficult to review on GitHub. Please split it into multiple lines, e.g.

```ts
.to.be.rejectedWith(
  'Navigational properties are not allowed in model data (model "Customer" property "orders"). ' + 
    'Please remove it or make sure the relation name is not the same as the property name in belongsTo relation.',
);
```

(It's just an example, I am not sure if that's enough to keep the line length under 80 chars.)

The same comment applies to most other changed lines too.